gatsby/src/pages/past-speakers.js

```diff
@@ -1,9 +1,10 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
Repurposing the "past speakers" page for all speakers for all years. There are links to choose which year.

Reply:
Nice, I'll close the old issue that wanted to add this feature.